opensubtitles/find.go

```diff
@@ -0,0 +1,123 @@
+package opensubtitles
+
+import (
+	"context"
+	"net/http"
+)
+
+// FindService provides access to the find related functions
+// in the OpenSubtitles API.
+//
+// OpenSubtitles API docs: https://www.opensubtitles.com/docs/api/html/index.htm#find
+type FindService service
+
+type FeatureOptions struct {
+	//Feature ID of the feature
+	ID string `url:"id,omitempty"`
+	//IMDB ID of the feature
+	ImdbID string `url:"imdb_id,omitempty"`
+	//TMDB ID of the feature
+	TmdbID string `url:"tmdb_id,omitempty"`
+}
+
+type Feature struct {
```

Comment:
Codacy found an issue: [exported type Feature should have comment or be unexported](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)